goreleaser.yml

```diff
@@ -0,0 +1,27 @@
+project_name: trivy-go-plugin
+builds:
+  -
+    main: main.go
+    binary: trivy-go-plugin
+    ldflags:
+      - -s -w
+      - "-extldflags '-static'"
+      - -X main.version={{.Version}}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+
+archives:
+  -
+    format: tar.gz
+    name_template: "{{.ProjectName}}_{{.Version}}_{{.Os}}-{{.Arch}}"
+    replacements:
+      linux: Linux
```

Comment:
archive replacements is deprecated now
https://goreleaser.com/deprecations/#archivesreplacements